AnkiHubSoftware/ankihub_addon
# [BUILD-365] Store login credentials in profile config

## Related issues

https://ankihub.atlassian.net/browse/BUILD-365

## Proposed changes

This PR makes the add-on store login credentials in Anki's profile config (`m.pm.profile`) in preparation for the AnkiHub sign-in integration.

## How to reproduce

- Log in to the add-on and run this code in the debug console to confirm the credentials are stored in the profile config:
```python
print(mw.pm.profile["ankiHubToken"], mw.pm.profile["ankiHubUsername"])
```
- Check `.private_config.json` to confirm the username and token are no longer stored there.
